docs/_tutorials/bert-finetuning.md

```diff
@@ -183,18 +184,46 @@ Note:
 1. `batch_size` is the maximum bath size of input data, all fine-tuning training data or prediction data shouldn't exceed this threshold, otherwise it will throw an exception. In the DeepSpeed configuration file micro batch size is defined as `train_micro_batch_size_per_gpu`, e.g. if it is set as 8 and prediction uses batch size of 12, we can use 12 as transformer kernel batch size, or using "--predict_batch_size" argument to set prediction batch size to 8 or a smaller number.
 2. `local_rank` in DeepSpeedTransformerConfig is used to assign the transformer kernel to the correct device. Since the model already runs set_device() before here, so does not need to be set here.
 
-For more details about the transformer kernel, please see [DeepSpeed Transformer Kernel](/transformer_kernel/) and [DeepSpeed Fast-Bert Training](/fast_bert/).
+For more details about the transformer kernel, please see [DeepSpeed Transformer Kernel](/_tutorials/transformer_kernel/) and [DeepSpeed Fast-Bert Training](/_posts/2020-05-28-fastest-bert-training/).
+
+### Increasing Micro-Batch Size
```

Comment:
Can we just call this section Tuning Micro-Batch Size?

Reply:
Sure!